ResearchSuite/ResearchSuiteTests/Codable Tests/ExampleDecodableTests.swift

```diff
@@ -2,6 +2,8 @@
 //  ExampleDecodableTests.swift
 //  ResearchSuiteTests
 //
+//  Copyright © 2017 Sage Bionetworks. All rights reserved.
```

Comment:
should this be 2018? or was this code actually written last year

Reply:
File added last year, but was missing the copyright